feathr_project/test/test_user_workspace/feathr_config.yaml

```diff
@@ -102,6 +102,7 @@ online_store:
     ssl_enabled: True
 
 feature_registry:
+  endpoint: "https://feathr-sql-registry.azurewebsites.net/api/v1"
```

Comment:
Also add comments on this (maybe in the doc here https://github.com/linkedin/feathr/blob/main/docs/how-to-guides/feathr-configuration-and-env.md) so end users know what's this about

Reply:
My suggestion is, since we're going to remove PurView client, we should also remove the following `purview` section.
In this case `feature_registry.endpoint` won't cause any confusion.